README.bot.md

```
@@ -0,0 +1,243 @@
+lark-cli bot start
+
+# 使用配置文件
+lark-cli bot start --config ~/.lark-cli/bot-config.yaml
+
+# 后台运行
+lark-cli bot start --daemon
+```
+
+### 4. 在飞书中使用
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)